index.html

```diff
@@ -4,9 +4,81 @@
 	<meta charset="UTF-8">
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<title>Document</title>
+	<title>Marc Raphael Portofolio</title>
+
+	<!--Fonts-->
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+
+	<!--css links-->
+	<link rel="stylesheet" href="./style.css">
 </head>
 <body>
-	
+	<!-- the page's header -->
+	<header class="pg-hdr">
+		<h3>John Doe</h3>
+		<button type="button" class="btn">
+			<img src="./images/buttons/hamburger.svg" alt="menu-pic" >
+		</button>
+	</header>
+
+	<!-- the main content -->
+	<main id="main">
+		<section id="sct1">
+			<div class="titles">
+				<h1>Laura Jong</h1>
+				<h2>Software Developer</h2>
+			</div>
+			<p class="para">
+				I can help you build a product,
+				feature or website Look through some of my work and experience!
+				If you like what you see and have a project you need coded, don’t hestiate to contact me.
+			</p>
+
+			<!-- Soft Skills -->
+			<div class="skills">
+				<ul class="competences cmps-1">
+          <li class="cmp-ttl"><h3>Languages</h3></li>
```

Comment:
- [ ] Each of the `li` tags should have an `a` tag within. Do this for all `li` tags

Example `<li class="cmp-ttl"><a href="#'><h3>Languages</h3></a></li>`

Reply:
yhis is not a link